### DISCUSSION TITLE
Add a colour to date on Table view if date has already passed

### DISCUSSION THREAD

**Discussion by fabianwurk** [filamentphp/filament#6487] [Help] [answered]
I have a date field in Table view called 'expiry_date' and I want to highlight it in red colour if the date has already passed. Is that possible?

```
Tables\Columns\TextColumn::make('expiry_date')
    ->date()
```

Thanks

**Comment by RX-77IT** [comment, marked as answer]
Out of curiosity I just tried this in my Resource page and it worked
```php
use Filament\Tables;
use Illuminate\Database\Eloquent\Model;

public static function table(Table $table): Table
    {
        return $table
            ->columns([
                Tables\Columns\TextColumn::make('expiry_date')
                    ->extraAttributes(function (?Model $record) {
                        return $record->expiry_date < now()
                            ? ['style' => 'background-color: #ff1111;border-radius: 10px;']
                            : [];
                    }),
            ]);
    }
```

I used these two documentation pages for reference:
https://filamentphp.com/docs/2.x/tables/columns/getting-started#custom-attributes
https://filamentphp.com/docs/2.x/forms/advanced#using-closure-customization

**Reply by fabianwurk** [reply]
You are a legend, that worked perfectly 🙌 thanks so so much @RX-77IT 😀

**Reply by emincer** [reply]
Anybody know if I can do something similar except set the color for the entire row, based on one column?
